test/udev-test.pl

```diff
@@ -1550,10 +1551,9 @@ sub run_test {
 print "$error errors occurred\n\n";
 
 # cleanup
-system("rm", "-rf", "$udev_dev");
 system("rm", "-rf", "$udev_run");
 system("umount", "$udev_tmpfs");
```

Comment:
`udev_setup()` now does the `umount` right at the top, i. e. it's a no-op (and will fail) for the first time, and otherwise clean up the mount from the previous test. But I think it's cleaner to keep the umount `$udev_tmpfs` here, so that after the test it's gone again.